diff --git a/chrome-wrapper b/chrome-wrapper
index 31025de..6d98698 100755
--- a/chrome-wrapper
+++ b/chrome-wrapper
@@ -7,8 +7,8 @@
 # Running Chromium via this script makes it possible to set Chromium as the
 # default browser directly out of a compile, without needing to package it.
 
-DESKTOP="chromium-devel"
-TITLE="Chromium"
+DESKTOP="chromium-nightly"
+TITLE="Chromium Nightly"
 
 usage() {
   echo "$0 [--gdb] [--help] [--man-page] [--] [chrome-options]"
